ovirtlago/cmd.py

```diff
@@ -114,6 +114,11 @@ def do_ovirt_runtest(prefix, test_file, **kwargs):
     type=os.path.abspath,
 )
 @cli_plugin_add_argument(
+    '--repo-path',
```

Comment:
I'd be a little bit more explicit, something like '--internal-repo-path' or '--prefix-repo-path'. There's another option that has already a bad naming just before '--rpm-repo' that makes this one confusing :confused: